src/uvw/stream.hpp

```diff
@@ -171,7 +171,9 @@ class StreamHandle: public Handle<T, U> {
     }
 
 protected:
-    using Handle<T, U>::Handle;
+	StreamHandle(std::shared_ptr<Loop> ref) noexcept
```

Comment:
Why can't we use the inherited constructor here? I mean, we use them all around the code (see `Handle` itself), why doesn't this one work? Are we solving the wrong problem? It goes without saying that inherited constructors are accepted even on MSVC (otherwise you shouldn't be able to create any handle), so what?